### PR TITLE
Adding the mount target props

### PR DESCRIPTION
**Pull request by OssamaZ**
I don't know if it is that useful or not, i had a case where specifying the mount target for the frame children was necessary.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,7 +78,15 @@ var Frame = React.createClass({
       // unstable_renderSubtreeIntoContainer allows us to pass this component as
       // the parent, which exposes context to any child components.
       var callback = initialRender ? this.props.contentDidMount : this.props.contentDidUpdate;
-      ReactDOM.unstable_renderSubtreeIntoContainer(this, contents, doc.body.children[0], callback);
+      var mountTarget;
+      
+      if(this.props.mountTarget) {
+        mountTarget = doc.querySelector(this.props.mountTarget);
+      }else {
```
Comment (ryanseddon) [inline index.js]: `}else {` -> `} else {`

File: test/Frame_spec.js [hunk 2]
```diff
@@ -204,6 +204,18 @@ describe("Frame test",function(){
...
+  it("should allow setting mountTarget", function () {
+    div = document.body.appendChild(document.createElement('div'));
+
+    var initialContent = '<!DOCTYPE html><html><head></head><body><h1>i was here first</h1><div id="mountHere"></div></body></html>';
+    var renderedContent = '<html><head></head><body><h1>i was here first</h1><div id="mountHere"><div data-reactid=".j"></div></div></body></html>';
+    var frame = ReactDOM.render(
+      <Frame initialContent={initialContent} mountTarget="#mountHere" />
+    , div);
+    var doc = ReactDOM.findDOMNode(frame).contentDocument;
+    expect(doc.documentElement.outerHTML).toEqual(renderedContent);
```
Comment (ryanseddon) [inline test/Frame_spec.js]: Add a new line above here.

File: test/Frame_spec.js [hunk 3]
```diff
@@ -204,6 +204,18 @@ describe("Frame test",function(){
     expect(doc.documentElement.outerHTML).toEqual(renderedContent);
   });
 
+  it("should allow setting mountTarget", function () {
+    div = document.body.appendChild(document.createElement('div'));
+
+    var initialContent = '<!DOCTYPE html><html><head></head><body><h1>i was here first</h1><div id="mountHere"></div></body></html>';
+    var renderedContent = '<html><head></head><body><h1>i was here first</h1><div id="mountHere"><div data-reactid=".j"></div></div></body></html>';
+    var frame = ReactDOM.render(
+      <Frame initialContent={initialContent} mountTarget="#mountHere" />
```
Comment (ryanseddon) [inline test/Frame_spec.js]: Add a child here since react 15 will drop the `data-reactid` attributes so you should check for something else.

**Comment by ryanseddon** [comment]
Nice I think this will be useful especially with setting `initialContent`

**Comment by OssamaZ** [comment]
Sorry for the grammar typo in the test ..

**Comment by ryanseddon** [comment]
Thanks @OssamaZ completely forgot about this PR feel free to ping me if you don't get a response.

**Comment by ryanseddon** [comment]
Just push v0.6.0

**Pull request merged**